.docker/php8.3-apache/Dockerfile

```diff
@@ -35,10 +35,15 @@ RUN apt-get update \
     && docker-php-ext-install intl pdo pdo_mysql mysqli \
     && docker-php-ext-configure gd --with-jpeg --with-freetype \
     && docker-php-ext-install gd zip \
-    && pecl install xdebug imagick \
-    && docker-php-ext-enable xdebug imagick \
+    && pecl install xdebug \
+    && docker-php-ext-enable xdebug \
     && rm -rf /var/lib/apt/lists/*
 
+# Install imagick using the mlocati helper script (alternative approach)
+ADD https://github.com/mlocati/docker-php-extension-installer/releases/latest/download/install-php-extensions /usr/local/bin/
```

Comment:
Personally never used this extension and kind of would like to avoid using unessecary layers as it might bring also vulnerability issues ... Is it really needed to have that layer? maybe i can fix this without an extra layer.